crates/rpc/rpc/src/eth/api/pending_block.rs

```diff
@@ -244,6 +245,15 @@ impl PendingBlockEnv {
         let blob_gas_used =
             if cfg.handler_cfg.spec_id >= SpecId::CANCUN { Some(sum_blob_gas_used) } else { None };
 
+        // note(onbjerg): the rpc spec has not been changed to include requests, so for now we just
+        // set these to empty
+        let (requests, requests_root) =
+            if chain_spec.is_prague_active_at_timestamp(block_env.timestamp.to::<u64>()) {
+                (Some(Requests::default()), Some(EMPTY_ROOT_HASH))
+            } else {
+                (None, None)
+            };
+
```

Comment:
https://github.com/ethereum/execution-apis/pull/545

Reply:
this is the engine api, not the json rpc